src/app/dim-ui/Sheet.tsx

```diff
@@ -251,23 +250,15 @@ export default function Sheet({
           <a
             href="#"
             className={clsx('sheet-close', { 'sheet-no-header': !header })}
-            onClick={handleClose}
+            onClick={triggerClose}
           >
             <AppIcon icon={disabledIcon} />
           </a>
 
-          <div
-            className="sheet-container"
-            onPointerDown={dragHandleDown}
-            onPointerUp={dragHandleUp}
-            onMouseDown={dragHandleDown}
-            onMouseUp={dragHandleUp}
-            onTouchStart={dragHandleDown}
-            onTouchEnd={dragHandleUp}
-          >
+          <div className="sheet-container" onPointerDown={dragHandleDown}>
```

Comment:
Framer only makes use of the pointer events for its drag functionality. This is also why the `touch-action: none` css property is set. See https://www.framer.com/docs/use-drag-controls/#touch-support